packages/loot-design/src/components/modals/ImportTransactions.js

```diff
@@ -880,6 +984,8 @@ export function ImportTransactions({
             Import {transactions.length} transactions
           </ButtonWithLoading>
         </View>
+
+        <View style={{ flex: 1 }} />
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏IMO this element is unnecessary (removed it from the DOM and the layout was still the same).

Reply:
Yep, cool. Removed it.